parser/unescape.go

```diff
@@ -146,9 +149,11 @@ func unescapeChar(s string) (value rune, multibyte bool, tail string, err error)
 	// 4. Unicode escape sequences, reproduced from `strconv/quote.go`
 	case 'x', 'X', 'u', 'U':
 		n := 0
+		encode = true
 		switch c {
 		case 'x', 'X':
 			n = 2
+			encode = !is_bytes
 		case 'u':
```

Comment:
Looks like Unicode encodings aren't supported for byte literals in Google SQL. 
https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#literals

Reply:
Fixed.